README.md

```diff
@@ -1,25 +1,110 @@
-[The Scala Refactoring Project](http://scala-refactoring.org)
-================================================================================
+# The Scala Refactoring Library
 
-Welcome to the Scala Refactoring project!
+The Scala Refactoring Library implements IDE independent refactoring support
+for Scala. It is currently used by both [ENSIME](https://github.com/ensime)
+and [ScalaIDE](http://scala-ide.org/) and supports Scala-2.10 and Scala-2.11.
 
-The project requires Scala 2.10 but also supports 2.11.
+## Information for Contributors
 
-1. Building
---------------------------------------------------------------------------------
+### Help Needed
 
-We use Maven to build the project:
+We are always looking for helping hands. As you may have already found out,
+there are quite a few bugs to work on. Pull requests are usually reviewed and
+merged quickly, and we try to help new contributors wherever we can. We are
+currently focusing our efforts on 
 
-```bash
-$ ./build-2.11.sh
-```
+* Rename
+* Organize Imports
+* Add Import
+* Move Class
+
+All other refactorings, like
+
+* Extract Method
+* Extract Value
+* Extract Trait
+* Merge Parameter Lists
+* Split Parameter Lists
+* Change Parameter Order
+
+are effectively unmaintained. If you are particularly fond of an operation from
+this list, this might be a very good place to start contributing!
+
+### Overall Architecture
+
+Refactorings are implemented on top of 
+[ASTs](http://docs.scala-lang.org/overviews/reflection/symbols-trees-types) 
+from the Scala Presentation Compiler. If you plan to work on the library, you
+should definitely take a look at
+[Mirko Sockers Master Theses](http://scala-refactoring.org/wp-content/uploads/scala-refactoring.pdf).
```

Comment:
I would just put it into another branch in the scala-refactoring repo. While doing so, we ca also move the tex sources to that branch, no need to keep them in the master branch.